guides/micronaut-jms-aws-sqs/kotlin/src/main/kotlin/example/micronaut/DemoController.kt

```diff
@@ -0,0 +1,13 @@
+package example.micronaut
+
+import io.micronaut.http.annotation.Controller
+import io.micronaut.http.annotation.Get
+
+@Controller  // <1>
+class DemoController(private val demoProducer: DemoProducer) {  // <2>
+
+    @Get("/demo")  // <3>
```

Comment:
The controller which has a side effect it publishes to a Queue. It should be mapped to POST instead of Get. 

I would annotate the method with `@Status(NO_CONENT)` and respond a 204 instead 200.